climatetree/Places-Microservice-Server
# Barebones

Working get api endpoint.

**Prereqs:**
- Postgres db instance setup.
- Database populated with db script in `main/resources`

**Steps to test**
1. Configure application.properties file to point to your local db
2. Run Maven clean install from IDE or terminal - `mvn clean install`
3. Start the spring boot app
4. Hit `http://localhost:8080/climatetree/places/70262` to see place json object in browser or post
